packages/infra/lib/ihe-stack.ts

```diff
@@ -170,12 +186,16 @@ export class IHEStack extends Stack {
       envVars: {
         MEDICAL_DOCUMENTS_BUCKET_NAME: props.config.medicalDocumentsBucketName,
         API_URL: props.config.loadBalancerDnsName,
+        ...(posthogSecretName ? { POST_HOG_API_KEY: posthogSecretName } : {}),
         ...(props.config.lambdasSentryDSN ? { SENTRY_DSN: props.config.lambdasSentryDSN } : {}),
       },
       vpc,
       alarmSnsAction,
       version: props.version,
     });
+
+    const posthogSecretKey = "POST_HOG_API_KEY";
```

Comment:
Why defining this multiple times instead of declaring a global const on this file?